users/services/identity.py

```diff
@@ -75,7 +82,7 @@ def mastodon_json_relationship(self, from_identity: Identity):
             "requested": False,
             "domain_blocking": False,
             "endorsed": False,
-            "note": "",
+            "note": follow.note or "",
```

Comment:
this is unrelated to show/hide boosts but it's another property of the follow object which was previously unavailable. I can split this into a separate PR if that would be cleaner.

Reply:
This is fine!